test/StakeManager.t.sol

```diff
@@ -245,6 +294,76 @@ contract ExecuteAccountTest is StakeManagerTest {
         vm.expectRevert(StakeManager.StakeManager__InvalidLimitEpoch.selector);
         stakeManager.executeAccount(address(userVault), currentEpoch + 1);
     }
+
+    function _createStakingAccount(
+        address owner,
+        uint256 amount,
+        uint256 lockTime
+    )
+        internal
+        returns (StakeVault userVault)
+    {
+        deal(stakeToken, owner, amount);
+        userVault = _createTestVault(owner);
+        vm.startPrank(owner);
+        ERC20(stakeToken).approve(address(userVault), amount);
+        userVault.stake(amount, lockTime);
+        vm.stopPrank();
+    }
+
+    function test_ExecuteAccountMintMP() public {
+        uint256 stakeAmount = 10_000_000;
+        deal(stakeToken, testUser, stakeAmount);
+
+        userVaults.push(_createStakingAccount(makeAddr("testUser"), stakeAmount, 0));
+        userVaults.push(_createStakingAccount(makeAddr("testUser2"), stakeAmount, 0));
+        userVaults.push(_createStakingAccount(makeAddr("testUser3"), stakeAmount, 0));
+
+        console.log("######### NOW", block.timestamp);
+        console.log("# START EPOCH", stakeManager.currentEpoch());
+        console.log("# PND_REWARDS", stakeManager.pendingReward());
+
+        for (uint256 i = 0; i < 3; i++) {
```

Comment:
```suggestion
        for (uint256 i = 0; i < userVaults.length; i++) {
```

Reply:
I just realized `i` isn't used anywhere, so I'm not sure this actually implied `userVaults.length`, since you're also using `userVaults.length` in the inner loop.

Is there any particular reason this is supposed to loop 3 times? If yes, can we add this via some constant or variable, so this isn't getting as confusing in the future?